llm-token-limits/llm_token_limits.ipynb

```diff
@@ -0,0 +1,284 @@
+        "* Plan for scalability: Forecast future demand and ensure resource availability.\n",
+        "\n",
+        "To use this dashboard, from the Apigee console navigate to `Custom Reports` > `Tokens Consumption Report`. You'll be able to drill down into token metrics that represent consumption by Developer Apps and Products. See sample below:\n",
+        "\n",
+        "![image](images/token-counts.png)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "NmRI6i4uhnJ4"
+      },
+      "source": [
+        "# 2000 tokens every 5 minutes\n",
+        "\n",
+        "This scenario demonstrates a basic interaction with a language model. The code repeatedly asks a language model the same question, \"Why is the sky blue?\" but phrased in different ways. It's a simple example of how to interact with a language model. After running the scenario **only onnce** expect the following behavior:\n",
+        "\n",
+        "* If using the **Bronze Key**, fhe final token count (sum of tokens from prompts and response candates) shouldn't exceed the Bronze AI Product tokens limit of 2000 tokens every 5 minutoes.\n",
+        "* If using the **Silver Key**, fhe final token count (sum of tokens from prompts and response candates) shouldn't exceed the Silver AI Product tokens limit of 5000 tokens every 5 minutoes.\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 6,
+      "metadata": {
+        "id": "87i610BzZns_"
+      },
+      "outputs": [],
+      "source": [
+        "prompts = [\"Why is the sky blue?\",\n",
+        "           \"What makes the sky blue?\",\n",
+        "           \"Why does the sky is blue colored?\",\n",
+        "           \"Can you explain why the sky is blue?\",\n",
+        "           \"The sky is blue, why is that?\"]\n",
+        "\n",
+        "def invoke_model(prompt, model_to_invoke=model):\n",
+        "  model_to_invoke.invoke(prompt)\n",
+        "\n",
+        "for prompt in prompts:\n",
+        "  model.invoke(prompt)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ZjW8UBEnZg9U"
+      },
+      "source": [
+        "# 5000 tokens every 5 minutes\n",
+        "\n",
+        "This scenario demonstrates a basic interaction with a language model. The code repeatedly asks a language model the same question, \"Why is the sky blue?\" but phrased in different ways to make sure the candidate responses are very extensive (high token count). After running the scenario **only onnce** expect the following behavior:\n",
+        "\n",
+        "* If using the **Bronze Key**, fhe final token count (sum of tokens from prompts and response candates) **should exceed** the Bronze AI Product tokens limit of 2000 tokens every 5 minutoes. SHoud should expect `HTTP 429` error messages in the notebook and also visible on Apigee's debug session.\n",
```

Comment:
`candates` -> `candidates`?
`minutoes` -> `minutes`
`SHoud should` -> `Should`